docs/modules/operation/pages/deep-dive/telemetryd/introduction.adoc

```diff
@@ -99,20 +125,26 @@ parsers.0.class-name=org.opennms.netmgt.telemetry.protocols.netflow.parser.Ipfix
 parsers.0.queue.use-routing-key=false
 ----
 
-== Push Sensor Data through Minion
-Listeners and their parsers may run on either {page-component-title} or Minion, whereas adapters run on {page-component-title} or Sentinel.
-If a listener and its parsers are running on Minion, the received messages will be dispatched automatically to the associated adapters running in {page-component-title} or Sentinel via a queue.
-Minions can listen (receive) data, but require {page-component-title} or Sentinel to process.
+== Push sensor data through Minion
+
+Listeners and their parsers may run either on {page-component-title} or a Minion, whereas adapters may run either on {page-component-title} or Sentinel.
+If a listener and its parsers are running on a Minion, the received messages are dispatched automatically via a queue to associated adapters running in {page-component-title} or Sentinel.
+Minions can receive data, but require {page-component-title} or Sentinel to process it.
 
 .Running Listener on a Minion forwarding packets using the messaging system
-image::telemetryd/telemetryd-minion.png[]
+image::telemetryd/telemetryd-minion.png["Network overview diagram showing a listener running on a Minion and its related components"]
```

Comment:
is it a listener running on minion and the Minion's related components, or a listener and its related components running on a Minion?

Reply:
I think it's the latter, in which case I'll update the wording.

```suggestion
image::telemetryd/telemetryd-minion.png["Network overview diagram showing a listener running on a Minion, and the listener's related components"]
```